src/aliasRequestApiClient.ts

```diff
@@ -0,0 +1,85 @@
+import { IAliasRequest, IAliasCallback } from "./identity.interfaces";
+import { MParticleWebSDK } from "./sdkRuntimeModels";
+import Constants from './constants';
+import { FetchUploader, XHRUploader } from './uploaders';
+
+
+var HTTPCodes = Constants.HTTPCodes,
+    Messages = Constants.Messages;
+
+interface IAliasResponseBody {
+    message?: string
+}
+
+export async function sendAliasRequest (mpInstance: MParticleWebSDK, aliasRequest: IAliasRequest, aliasCallback: IAliasCallback) {
+        const { verbose, error } = mpInstance.Logger;
+        const { invokeAliasCallback } = mpInstance._Helpers;
+        const { aliasUrl } = mpInstance._Store.SDKConfig;
+        const { devToken } = mpInstance._Store;
+
+        verbose(Messages.InformationMessages.SendAliasHttp);
+
+        const uploadUrl = `https://${aliasUrl}${devToken}/Alias`;
```

Comment:
it looks like it doesn't need to be capitalized, I will add a comment saying as such, but in order to reduce total code changes, i am going to keep it capitalized since it was previously.